tests/service/CompletionTests.fs

```diff
@@ -80,4 +80,11 @@ let ``Underscore dot lambda - method completion`` () =
 let myFancyFunc (x:string) = 
     x 
     |> _.ToL"""
-    assertHasItemWithNames ["ToLower"] info
\ No newline at end of file
+    assertHasItemWithNames ["ToLower"] info
+
+[<Test>]
+let ``Type decl - Record - Field type 01`` () =
+    let info = getCompletionInfo "type Record = { Field:  }" (2, 23)  """
+type Record = { Field:  }
+"""
+    assertHasItemWithNames ["string"] info
```

Comment:
Could be also anything else here (int, float, whatever qualifies) - right?

Reply:
Yes, it basically just checks that FCS provides some useful items here.

Reply:
Yep, just wanted to doublecheck, thx